facilitator/src/aggregation.rs

```diff
@@ -157,9 +159,9 @@ impl<'a> BatchAggregator<'a> {
         let mut included_batch_uuids = Vec::new();
 
         let mut servers = None;
-        let mut ingestion_header = None;
+        let mut ingestion_header: Option<IngestionHeader> = None;
```

Comment:
nit: could we rename this to last_ingestion_header?

Reply:
Sure. (though this could also be called `arbitrary_ingestion_header` -- the implementation currently chooses the last one, but we could just as easily use *any* of the ingestion headers we've seen. the expectation is that the fields we read match for all of the ingestion headers, which is now validated with this PR)